static/app/views/starfish/views/screens/screenLoadSpans/metricsRibbon.tsx

```diff
@@ -1,9 +1,10 @@
 import {useMemo} from 'react';
```

Comment:
nit: ha, for small enough components like this one, i actually don't mind the duplicated code, especially if it makes for faster iteration! it's not too much duplicated code, the [list](https://github.com/getsentry/sentry/pull/63191/files?diff=split&w=1#diff-d0021cf60157ca6d8e6aca3aed5df1d1cc77cf554d5e7a1bb3fa10aa980935c5R127) of fields and metadata we end up abstracting out ends up being a long list anyway and ends up including that dataKey callback which in some cases is doing a little more stuff and it'd be nice to just see that transformation on the component rendering that data.